### ISSUE TITLE
Size of blocks and fonts in Blockly

### ISSUE THREAD

**Issue by motib** [Mobsya/blockly#9]
With computer screens having ever-higher resolutions, it is important that the blocks and fonts in the blocks either automatically adapt to the resolution or allow the user to choose the size. On a laptop with 2560x1440, the blocks were close to unreadable.

**Comment by stephanemagnenat** [comment]
Agreed, and this is a problem with this version of Blockly. As a temporary hack, we can imagine to zoom the web view, if this is available. Could you also open a similar issue in Aseba, to track the possible zoom trick there?
